### ISSUE TITLE
Parallel load and run in MySQL cluster

### ISSUE THREAD

**Issue by ssizmaster** [brianfrankcooper/YCSB#1277]
Hi,
I would like to know how to load data and run  Workloads A and C in parallel in MySQL NDB  Cluster with two nodes?
Thank you in advance.

**Comment by busbey** [comment]
closing as duplicate of #1311
